mono_repo/lib/src/commands/pub.dart

```diff
@@ -41,17 +41,29 @@ class _PubSubCommand extends MonoRepoCommand {
 }
 
 Future<Null> pub(String pubCommand, RootConfig rootConfig) async {
-  var pkgDirs = rootConfig.map((pc) => pc.relativePath).toList();
-
   print(lightBlue
-      .wrap('Running `pub $pubCommand` across ${pkgDirs.length} package(s).'));
+      // `pub` isn't really a fitting name anymore. What do we use?
```

Comment:
We should move the `executable` check above this and then use executable + arguments in the content of the log

Reply:
Put a more generic message outside the loop, then include the detailed command in the "Starting `$dir`... bit...